runestone/activecode/js/activecode.js

```diff
@@ -437,7 +437,7 @@ ActiveCode.prototype.createOutput = function () {
 
 
     var lensDiv = document.createElement("div");
-    $(lensDiv).addClass("col-md-6");
+    $(lensDiv).addClass("col-md-12");
```

Comment:
This was needed for presenter view and we don't think it hurts anything in regular textbook mode.